### ISSUE TITLE
64 packages have newer versions incompatible with dependency constraints. Try `flutter pub outdated` for more information &  Error: 'getTranslationListFirestoreData' is exported from both 'package:menuz/backend/schema/structs/translation_list_struct.dart' and 'package:menuz/backend/schema/structs/translation_struct.dart'.

### ISSUE THREAD

**Issue by ogatayosuke0117** [FlutterFlow/flutterflow-issues#3474]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

<img width="362" alt="スクリーンショット 2024-07-25 140856" src="https://github.com/user-attachments/assets/2634980c-ff65-4ae1-801d-932410cc4793">

### Expected Behavior

Solve it or give me solution

### Steps to Reproduce

all in image
I installed firebase translation extention 1 week ago, that might be issue?? this one! 
https://extensions.dev/extensions/firebase/firestore-translate-text

and also, traslation in flutterflow is activated at same time, with multiple languages, 

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITEei/HlvM5Jn7wG0Kn6K8JonQMwJ0QlUIInj+0bWRMhfar2PIoieePObRdLYteYfwlqHFakhXMH0c7rkYbxLPA5ORaZbqJz1r5XQTmUezm7WrGbDZCCbXF7EZhRIVSx4Ym0kBMmJtBiSU4b706YJfCdG3qCf9qOYwx5e6fDbOY=

### Visual documentation

<img width="362" alt="スクリーンショット 2024-07-25 140856" src="https://github.com/user-attachments/assets/2634980c-ff65-4ae1-801d-932410cc4793">

### Environment

```markdown
- FlutterFlow version: FlutterFlow v4.1.76+ released July 25, 2024
Flutter version is 3.22.2
- Platform:Web
- Browser name and version:Chrome 
- Operating system and version affected: Windows11
```

**Comment by ogatayosuke0117** [comment]
Project ID ; menuz-zu0vj4

**Comment by Alezanello** [comment]
Hello!

The issue seems to be that the error log indicates a conflict with the translation of Firestore data from two packages:

`Error: 'getTranslationListFirestoreData' is exported from both 'package:menuz/backend/schema/structs/translation_list_struct.dart' and 'package:menuz/backend/schema/structs/translation_struct.dart'.`

Try removing one of these conflicting exports to see if it resolves the issue.

Best regards,<br>Azanello

**Comment by ogatayosuke0117** [comment]
how can I see the 2 darts code?? I removed extention from firebase though
'package:menuz/backend/schema/structs/translation_list_struct.dart' and 'package:menuz/backend/schema/structs/translation_struct.dart'

**Comment by ogatayosuke0117** [comment]
I mean how can I remove of them?

**Comment by ambience-kim** [comment]
I also suddenly started having similar issues about 2 hours ago. (See attached image)

For reference, before this issue occurred, I was modifying the UI and backend logic, and the compilation was fine until this issue occurred.
And when I tried to check the result after changing the label value on the screen, the compilation error started occurring.

The strange thing is, I have created and used several custom widgets, but I have not created any custom widgets in the past 2 weeks.

![스크린샷 2024-08-01 오후 6 18 35](https://github.com/user-attachments/assets/304d9ef6-513b-42ef-bce9-d9bdec176ac7)

**Comment by ambience-kim** [comment]
I tried to fix this error yesterday by excluding my custom widgets from compilation. But this error still occurred, so I gave up working on FF yesterday.

And today I didn't do anything, but this issue disappeared. So I included all my custom widgets in the project, and this issue still doesn't occur.

That means this issue doesn't occur to me now, and this issue disappeared regardless of my work.
